clean.sh

```diff
@@ -9,8 +9,12 @@ for version; do
     if [ -f "$version"/"$id_file" ]; then
```

Comment:
"$version"/"$id_file" is used three times. What about to use a new variable.

Small nitpicking.

Reply:
You are right.